### ISSUE TITLE
Hurricane model start time defined by input file

### ISSUE THREAD

**Issue by JaroCamphuijsen** [omuse-geoscience/omuse#73]
Currently the HollandHurricane model (src/omuse/ext/hurricane_models.py) always starts from the start (top line) of the hurricane track file that is provided at initialization. This results in unexpected/unwanted behavior when it is coupled to a different model which does involve an explicit start date. The hurricane model will then act as if the hurricane started at the start time of the other model (since model_time is 0). I suggest to offer an option to select the start time such that it does not depend on the provided hurricane track data. 

This requires:
- Adding "start_time" as an optional argument for the initialization function
- Change `initialize_from_track` to take this start date as `t0` if it is provided
- Raise an issue when the provided start date is out of bounds of the provided track file times

**Comment by JaroCamphuijsen** [comment]
Meanwhile the evolve function should also raise an issue if a simulation step is requested which is beyond the last value of the provided track file. Currently it will just keep returning the last value indefinitely, as if the hurricane had stopped in its track. I believe this is also unexpected/unwanted behavior and at least something to be notified about.

**Comment by esclapez** [comment]
@JaroCamphuijsen If I'm not mistaken your last point is not addressed in #74 right ? Can I go ahead and move forward on that PR or do you want to include this last feature as well ?

**Comment by JaroCamphuijsen** [comment]
@esclapez  yeah, thanks, for the MOSAIC project it would be good to already merge #74. After October I can work on the error message (we have to think about that though because it would introduce breaking changes). I removed the "fix" mention of this issue so it stays open once you merge it and we can keep the branch open as well so we can continue the work on that.
